src/display/api.js

```diff
@@ -1181,8 +1182,7 @@ class PDFDocumentProxy {
  * Page render parameters.
  *
  * @typedef {Object} RenderParameters
- * @property {CanvasRenderingContext2D} canvasContext - A 2D context of a DOM
- *   Canvas object.
+ * @property {HTMLCanvasElement} canvas - A DOM Canvas object.
```

Comment:
nit: it could be an OffscreenCanvas too, right ?

Reply:
In theory, yes. At the moment there's two main parts of the lifecycle of the canvas. There's a part where changes are made assuming an `HTMLCanvasElement`, in this part we're still passing an element. Later on the canvas is transferred into an `OffscreenCanvas` and now I'm working on dispatching operations that can be made on that to the worker (since the canvas is no longer owned by the main thread).